ui/panels-plugin/src/plugins/line-chart/tooltip/TooltipContent.tsx

```diff
@@ -11,39 +11,64 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Box, Typography } from '@mui/material';
+import { Box, Divider, Stack, Typography } from '@mui/material';
 import { FocusedSeriesArray } from '../utils/focused-series';
 import SeriesInfo from './SeriesInfo';
 
-function TooltipContent(props: { focusedSeries: FocusedSeriesArray }) {
-  const { focusedSeries } = props;
-  let lastDate = focusedSeries[0] && focusedSeries[0].date ? focusedSeries[0].date : '';
+function TooltipContent(props: { focusedSeries: FocusedSeriesArray; wrapLabels?: boolean }) {
+  const { focusedSeries, wrapLabels } = props;
+  // let lastDate = focusedSeries[0] && focusedSeries[0].date ? focusedSeries[0].date : '';
+  const seriesTime = focusedSeries[0] && focusedSeries[0].date ? focusedSeries[0].date : false;
+
+  const formatTimeSeriesHeader = (timeString: string) => {
+    console.log('time string is', timeString);
+    const [month, year, time] = timeString.split(',');
+    return (
+      <>
+        <Typography variant="caption" color="grey.300">
+          {month}, {year} –
+        </Typography>
+        <Typography variant="caption">
+          <strong>{time}</strong>
+        </Typography>
+      </>
+    );
+  };
+
   return (
-    <>
-      {focusedSeries.map(({ datumIdx, seriesIdx, seriesName, date, y, markerColor }, index) => {
-        if (datumIdx === null || seriesIdx === null) return null;
-        const key = seriesIdx.toString() + datumIdx.toString();
+    <Stack py={1} px={1.5} spacing={0.5}>
+      {seriesTime && <Typography variant="caption">{formatTimeSeriesHeader(seriesTime)}</Typography>}
+      <Divider sx={{ borderColor: 'grey.800' }} />
+      <Box
+        sx={{
+          display: 'table',
+        }}
+      >
+        {focusedSeries.map(({ datumIdx, seriesIdx, seriesName, y, markerColor }) => {
+          if (datumIdx === null || seriesIdx === null) return null;
+          const key = seriesIdx.toString() + datumIdx.toString();
 
```

Comment:
@sjcobb I still need to resolve this commented out section, my goal here was to keep all the `SeriesInfo` elements grouped together to have more consistent display of them, but I am not sure if my approach is the best. Would love any feedback here.

Reply:
For context, one thing I had been playing around with was increasing xBuffer to show more nearby points. So potentially multiple dates could show in the tooltip (which required sorting differently). I think that is overkill though so all the lastDate stuff can probably be removed.

Reply:
Is it something that should be solved before merging this PR, or we could keep it for later ?

Reply:
Yeah it looks mostly done already, mainly just removing the commented out block